forc/src/ops/forc_contract_id.rs

```diff
@@ -0,0 +1,75 @@
+use crate::cli::ContractIdCommand;
+use anyhow::{bail, Result};
+use forc_pkg::{self as pkg, build_with_options};
+use sway_core::{fuel_prelude::fuel_tx, BuildTarget};
+use tracing::info;
+
+pub fn contract_id(command: ContractIdCommand) -> Result<()> {
+    let build_options = build_opts_from_cmd(&command);
+    let build_plan = pkg::BuildPlan::from_build_opts(&build_options)?;
+    // If a salt was specified but we have more than one member to build, there
```

Comment:
We might want to create a new issue to handle multiple salts when https://github.com/FuelLabs/sway/pull/4362 goes in, but for now this seems OK to me

Reply:
Created #4338.